README.md

```diff
@@ -43,6 +43,8 @@ Alternatively, you can also install it locally by installing the development hea
 $ sudo apt-get install zlib1g-dev libbz2-dev liblzma-dev clang pkg-config
 ```
 
+Refer to the `docker` directory in this repository for more up to date local instructions, including LLVM installation.
```

Comment:
```suggestion
We provide Dockerfile bases that provide these dependencies. Refer to the [docker](https://github.com/rust-bio/rust-htslib/tree/master/docker) directory in this repository for the latest instructions, including LLVM installation.
```
I changed "more up to date" to "latest" because the first implies "these instructions aren't up to date" and the latter doesn't say anything about the instructions here, but implies the instructions in the repository indeed are up to date.